### PR TITLE
common: Turn remote camera off/ask to turn camera on

### PR DESCRIPTION
**Pull request by thyal**
### Description

Add the possibility for hosts to turn off cameras for remote participants. The action turnOffParticipantCameras takes an array of clientIds, so you can also implement "turn off all cameras" functionality with this. Also adds `askToTurnOnCamera` function, that a host can use to ask a participant to enable their camera.

**Summary:**

**Related Issue:**

### Testing

1. Check out this branch
2. In the /.env file add a valid host roomKey for the configured STORYBOOK_ROOM in following line:
STORYBOOK_ROOM_HOST_ROOMKEY=[host_roomKey_here]
3. Run yarn build && yarn dev
4. In the Storybook UI (@ http://localhost:6006/) open the story: "Custom UI" > "Room Connection With Host Controls"
5. Join the same room in a different browser (PWA).
6. Clicking on the turn off camera button on the remote participant should turn the camera off in the other tab (PWA side). 
image
7. Verify that it in fact does turn the camera off, you should see this notification in the PWA tab:
image
8. Test the other direction -> verify that turning cam off from the PWA does turn off in the SDK (and a notification will be shown in storybook).
9. Additionally, this can also be tested as a non-host (remove the host key from .env for example) - the actions should not work. 

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [x] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/core/src/redux/slices/notifications/index.ts [hunk 1]
```diff
@@ -157,6 +157,36 @@ startAppListening({
...
+            return;
+        }
+
+        dispatch(
+            doSetNotification(
+                createNotificationEvent<"requestVideoEnable" | "requestVideoDisable", RequestAudioEventProps>({
+                    type: enable ? "requestVideoEnable" : "requestVideoDisable",
+                    message: enable
+                        ? `${client.displayName} has requested for you to start video`
+                        : `${client.displayName} has stopped your video`,
```
Comment (richtrwhereby) [inline packages/core/src/redux/slices/notifications/index.ts:179]: Should we reflect the message that we show in PWA here. i.e. `${client.displayName} has stopped your video. You can turn your camera back on any time.`. Or perhaps we should just display this message at the demo app toast level and not change this simple messaging here.

File: packages/browser-sdk/src/stories/components/VideoExperience.tsx [hunk 2]
```diff
@@ -120,6 +123,37 @@ export default function VideoExperience({
...
 
+    function showRequestVideoEnableNotification({ message }: RequestVideoEvent) {
+        toast(
+            (t) => (
+                <div>
+                    {message}
+                    <div>
+                        <button
+                            onClick={() => {
+                                toggleCamera(true);
```
Comment (richtrwhereby) [inline packages/browser-sdk/src/stories/components/VideoExperience.tsx:134]: With the SDK camera off, this action did not work for me when I requested the SDK user to turn on their camera from a PWA host URL. I did see the toast notification correctly in the SDK but clicking on `Start video` button only resulted in the following error in the console: `Action cannot be performed outside of a connected room`

**Review by richtrwhereby — CHANGES_REQUESTED** [review]

File: packages/browser-sdk/src/stories/components/VideoExperience.tsx (hunk 2 above)
Comment (thyal) [reply packages/browser-sdk/src/stories/components/VideoExperience.tsx:134]: oh. nice catch. I've just tested now and it's actually broken for microphone as well. Need to dig a bit deeper to see whats going on.

File: packages/core/src/redux/slices/notifications/index.ts (hunk 1 above)
Comment (thyal) [reply packages/core/src/redux/slices/notifications/index.ts:179]: I was just following the pattern from the mute/unmute notification (which also is different from the PWA). Happy to change it, but then I think we should change both.

File: packages/browser-sdk/src/stories/components/VideoExperience.tsx (hunk 2 above)
Comment (richtrwhereby) [reply packages/browser-sdk/src/stories/components/VideoExperience.tsx:134]: This is now fixed in the last commit by moving the `roomConnection.status === "connected"` check in to the `core` package (away from `useRoomConnection` ⭐

File: packages/core/src/redux/slices/notifications/index.ts (hunk 1 above)
Comment (richtrwhereby) [reply packages/core/src/redux/slices/notifications/index.ts:179]: I don't have a strong opinion on this so let's leave it for now.

**Review by richtrwhereby — APPROVED** [review]
Tested and this works great 🎉

**Pull request merged**